### ISSUE TITLE
M2 macOS unlimited FPS problem.

### ISSUE THREAD

**Issue by lilwayne71** [haxball/haxball-issues#1980]
Hello basro,

First of all, thank you for your new updates. We no longer need any js codes, add-ons, clients. You made it great.

Basro, I've **intel** chip macOS and I'm playing the game for years without any big issue, even after new update. My friends has just bought **m2** chip macOS, as you said in haxball news we don't write d3d9 code for fps unlock. Just disable frame limit.

open -a 'Google Chrome' --args —disable-gpu-vsync --disable-frame-rate-limit

After writing it to Terminal to open browser uncapped fps, I've no problem with this code in my system. However in my friends' pc, right after the green pitch has come, his screen just freeze and stop to work with all browsers you can imagine. Also there is no client for **m2** macOS. What should my friend do to play with uncapped fps?

We also try to d3d9 code but as you can imagine, it lowers fps excessively that can not let play in macOS.

**Comment by basro** [comment]
d3d9 will not work on macOS since it means DirectX 9 which is a windows specific thing.

Unfortunately I don't own a mac so I have no way of investigating it. You could try going to chrome://flags and looking for the angle backend setting (which is what you are setting with the --angle command line flag). There might be other options available in macos that you can try.

**Comment by lilwayne71** [comment]
Yep I know d3d9 is a windows thing, but I gave it try :). That command line says it is default and there are two options as OpenGL and Metal. Is there anything to do?

I have intel chip macOs he has m2 chip. I can play unlimited fps however he can’t. We are professional players in same team and we need a solution :( I got you you don’t have any idea but maybe you can suggest a possible solution..

**Comment by swizzhax** [comment]
@lilwayne71 

Playing on MBP with M2 Pro so far with configuration as below on Electron and everything is working fine, fps is unlocked.

```
app.commandLine.appendSwitch("disable-frame-rate-limit");
app.commandLine.appendSwitch("disable-gpu-sandbox");
app.commandLine.appendSwitch('max-gum-fps="500');
```

**Comment by lilwayne71** [comment]
> @lilwayne71
> 
> Playing on MBP with M2 Pro so far with configuration as below on Electron and everything is working fine, fps is unlocked.
> 
> ```
> app.commandLine.appendSwitch("disable-frame-rate-limit");
> app.commandLine.appendSwitch("disable-gpu-sandbox");
> app.commandLine.appendSwitch('max-gum-fps="500');
> ```

Appreciate your help and answer. But we are not have any knowledge about node.js, codes etc. How can we install Electron and write these codes on it?
